.github/utils/update_version.py

```diff
@@ -91,10 +91,16 @@ def generate_release_notes(new_version: str) -> None:
             exit(1)
         versioned_release_notes = []
         for line in release_notes.splitlines():
-            if not ("unreleased" in line.lower() and "**" in line):
+            if not ("unreleased" in line.lower() and "**" in line) and line != "":
```

Comment:
Do we want to also filter out lines which contain only whitespace?
```suggestion
            if line.strip() and not ("unreleased" in line.lower() and "**" in line):
```